order.go

```diff
@@ -137,8 +145,8 @@ type LineItem struct {
 	Grams                      int              `json:"grams"`
 	FulfillmentStatus          string           `json:"fulfillment_status"`
 	TaxLines                   []TaxLine        `json:"tax_lines"`
-	OriginLocation             Address          `json:"origin_location"`
-	DestinationLocation        Address          `json:"destination_location"`
+	OriginLocation             *Address         `json:"origin_location"`
+	DestinationLocation        *Address         `json:"destination_location"`
```

Comment:
This is potentially a breaking change for some users. Have you noticed any issues with this *not* being a pointer?

Reply:
Well i didn't have any issue with this being not a pointer..
I think the right why in Go is to use it as a pointer, and i also tried to change it into pointer in order to stick to the convention the `Address` properties are being used inside the `Order` struct.

@dlebech  From my side I really don't mind to use it the way it was before. Let me know if you wish to keep using it as it was.

Reply:
@RanAbram I think that makes sense, it's also easier to check if the object is "empty" by checking for nil. Thanks for sharing your thoughts.